assignments/closure.js

```diff
@@ -1,15 +1,32 @@
 // ==== Challenge 1: Write your own closure ====
-// Write a simple closure of your own creation.  Keep it simple!
+const divisibleBy = (x) => {
+  return checkDivisibility = (y) => {
+    return (y % x === 0);
+  } 
+}
 
+let divByFive = divisibleBy(5);
+
+divByFive(100); //returns true
```

Comment:
You can also call it as `divisibleBy(5)(100)` instead of calling them separately.